orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -60,10 +58,11 @@ class DependentPipelineStarter implements ApplicationContextAware {
     User principal = getUser(parentPipeline)
 
     pipelineConfig.trigger = [
-      type                  : "pipeline",
-      user                  : principal?.username ?: user ?: "[anonymous]",
-      parentExecution       : parentPipeline,
-      parentPipelineStageId : parentPipelineStageId
+      type                 : "pipeline",
+      user                 : principal?.username ?: user ?: "[anonymous]",
+      parentExecution      : parentPipeline,
+      parentPipelineStageId: parentPipelineStageId,
+      parameters           : [:]
```

Comment:
without this subsequent lines NPE when they try to reference `pipelineConfig.trigger.parameters`